doc/rfd/0000/README.md

```diff
@@ -0,0 +1,110 @@
+---
+authors: Jaime Gomes <jaime.gomes@suse.com>
+state: discussion
+---
+
+# Request for Discussion
+
+The goal is to have an constructive discussion within the team about an idea and capture it in a
+document known by RFD.
+
+## Scope
+
+## Metadata
+
+1. authors: owners of the RFD must be listed with their name and email address 
+(John Doe <john.doe@wonder.land>).
+2. state: one of the states discussed below.
+
+## State
+
+1. draft
+2. discussion
+3. published
+4. committed
+5. abandoned
+
+The ```draft``` state indicates that the work is not ready for discussion and that a placeholder
+was set. The RFD owner is putting his thoughts in place before promoting to ```discussion```.
+
+Documents under active discussion within the team should be in ```discussion``` state. When reaching
```

Comment:
I am concerned by the use of `should be` here, and in the following state descriptions.
Here it leaves open the possibility of actively discussing a document which is still in `draft` state.
Similar for the other states.
And that means (I believe) what the various states are used for becomes a bit smeared in the mental landscape of the participants.

I will now try to come up with an alternate proposal for phrasing and describing the states where we use `must be` or `has to be` to proper delineate the differences between states.

Maybe this will even help with Mark's confusion. No promises though. As I may be confused as well, no ?

Reply:
Here is my rewrite of this entire section:

```
The state ```draft``` indicates that the work is not ready for discussion yet.
It is a placeholder for the RFD owner to put his thoughts into.
The RFD owner has to state the scope of the RFD as clearly and concise as possible before promoting it to ```discussion```.

A work in the ```draft``` state may be exposed to people other than the owner.

The state ```discussion``` indicates that the work is ready for sharing and discussion.
When reaching this state the scope of the RFD must be clear and well defined.
At that point the RFD must be exposed to people other than the owner.
This should be at least the team itself, and must include relevant external stakeholders as well.

The state ```published``` indicates that the discussion has completed, and converged to a clear understanding of what needs to be done.
At that point the RFD is official and has to be made visible in the main development branch of the relevant repository.

From then on minor comments must be raised as issues of the repository.
Comments making fundamental changes to the RFD however have to restart the RFD process with a new RFD intended to supercede the RFD they comment on.

The last state, ```abandoned```, indicates that the RFD will never be implemented.
This state and can be reached from all previous states.
For a draft RFD the owner makes the descision to abandon it.
For an RFD under discussion the owner makes the descision, with guidance from the participants.
This should especially be considered when the discussion does not converge, indicating fundamental problems in the RFD, be it scope or understanding.
For a published RFD higher levels in and above the team make the decision.
```

Note, I left ```committed``` as I did not understand the difference to ```published```, even with the latest explanation from @fargozhu . Will comment there also.

Reply:
I did add some of your comments into the section but some I left 'cause I would prefer to avoid "mechanization" of the process.

Reply:
I was less thinking `mechanization` and more `formal speaking`.
Looking ... Looks good. ... Actually like individual vs collective decision. Describes the difference without being strict on the collective.